config/initializers/assets.rb

```diff
@@ -8,6 +8,7 @@
 Rails.application.config.assets.precompile += %w(
```

Comment:
do we still need this file with https://github.com/openstax/accounts/blob/refactor_controllers/app/assets/config/manifest.js?

Reply:
The manifest is used by sprockets 4+ but we still have sprockets 3 in the Gemfile